Client/gui/CGUI_Impl.cpp

```diff
@@ -115,15 +121,23 @@ void CGUI_Impl::SetSkin(const char* szName)
         CEGUI::SchemeManager::getSingleton().unloadScheme(m_CurrentSchemeName);
     }
 
-    PushGuiWorkingDirectory(CalcMTASAPath(PathJoin("skins", szName)));
+    bool bConvertSkin = ConvertToModernSkin(szName);
+
+    if (!bConvertSkin)
+    {
+        BrowseToSolution("gui-skin", EXIT_GAME_FIRST, SString("Error converting skin '%s' to modern", szName));
```

Comment:
Future note: we have to create this help page in the future when this gets merged.

Reply:
We have a new repository for this now: https://github.com/multitheftauto/help.multitheftauto.com